src/vcpkg-utils.ts

```diff
@@ -116,6 +116,11 @@ export class Utils {
 
     key += "-args=" + Utils.hashCode(core.getInput(runvcpkglib.vcpkgArguments));
     key += "-os=" + Utils.hashCode(process.platform);
+
+    if (process.env.ImageVersion) {
+      key += "-imageVer=" + Utils.hashCode(process.env.ImageVersion);
```

Comment:
Should `IMAGE_VERSION` be used instead, given [this context](https://github.com/actions/virtual-environments/pull/2509)?

Reply:
Ah, I'd missed that - yes, I suspect so.

Reply:
I've switched this to use ```IMAGE_VERSION```, and have also added a preference for ```IMAGE_OS``` instead of ```process.platform``` in the "os" key.

Reply:
Hm, no, I think those variables are only used when configuring the VMs, but are not accessible at “runtime”. ImageVersion / ImageOs seem to be the intended variables to use. I’ll fix this again.

Reply:
The only side-effect of these changes is that the existing cached artifacts are all invalidated. This means I will publish a `run-vcpkg@v7`, i.e. bumpoing the _major version_, as this going to force existing workflows to regenerate all cached artifacts.